source/common/router/rds_impl.cc

```diff
@@ -56,7 +56,7 @@ StaticRouteConfigProviderImpl::~StaticRouteConfigProviderImpl() {
 // initialization needs to be fixed.
 RdsRouteConfigSubscription::RdsRouteConfigSubscription(
     const envoy::config::filter::network::http_connection_manager::v2::Rds& rds,
-    const std::string& manager_identifier, Server::Configuration::FactoryContext& factory_context,
+    const uint64_t& manager_identifier, Server::Configuration::FactoryContext& factory_context,
```

Comment:
I think you need to drop `&` here (and in the header file).

Reply:
done